crates/solvers/src/infra/dex/balancer/mod.rs

```diff
@@ -44,32 +44,13 @@ impl Sor {
         }
     }
 
-    async fn quote(&self, query: &dto::Query) -> Result<dto::Quote, Error> {
```

Comment:
I think it's fine to move code around if it makes more sense in the end but it would be nice to have refactors unrelated to the main change in a separate commit. Not a requirement, just nice.

Reply:
I find it easier to read if the `pub` methods come first. I can undo these changes if you want.

Reply:
You don't have to change the PR but then I would like it if the team could agree on some basic rules that would minimize the need to move around code to preserve the git history as much as possible.

Reply:
One advantage to "allowing" refactors and moving code around within a PR is that it makes it more convenient to refactor the core logic of the services (and removes additional barriers to keeping the abstractions "up to date", no need to go through the hassle for creating additional PRs for example).

As @MartinquaXD suggested, maybe we can come up with some "standard practices" for code ordering to make these less common?

Reply:
I just play it by ear for the most part. I want everybody to feel encouraged to constantly refactor the code, otherwise the code will rot as you add new features. This inevitably hurts git blame but it's worth it.